scripts/build_sdk.sh

```
@@ -0,0 +1,88 @@
+if [ -d "$SDK_OUTPUT_DIR" ]; then
+  echo "🧹 Cleaning up previous SDK..."
+  rm -rf "$SDK_OUTPUT_DIR"
+fi
+
+# Generate the SDK using OpenAPI Generator
+echo "📦 Generating TypeScript client..."
+npx openapi-generator-cli generate \
+  -i "$OPENAPI_SPEC" \
+  -g typescript-fetch \
+  -o "$SDK_OUTPUT_DIR" \
+  --additional-properties=npmName=$PACKAGE_NAME,npmVersion=$PACKAGE_VERSION,ensureUniqueParams=true,supportsES6=true,withInterfaces=true,importFileExtension=.js
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)